oesukam/andela-sendit
# #162082797 Corrected the API according to the LFA's feedback

#### What does this PR do?
Applies LFA's feedback related to the API

#### Description of Task to be completed?
- Correct GET ```/api/v1/users/userId/parcels``` endpoint
- Rename ```/api/v1/users/login``` endpoint to ```/api/v1/auth/login```
- Rename ```/api/v1/users``` andpoint to ```/api/v1/auth/signup```
- Move controllers' tests into routes-controllers directory
- Add the pull request template

#### How should this be manually tested?
Using Postman, you should be able to use the following endpoints:
- ```/api/v1/auth/login```: Login
- ```/api/v1/auth/signup```: Signup

When creating the next pull request, the description body should come from the PULL_REQUEST_TEMPLATE.md file

#### What are the relevant pivotal tracker stories?
#162082797
